cachito/errors.py

```diff
@@ -15,3 +15,7 @@ class ConfigError(CachitoError):
 
 class ContentManifestError(CachitoError, ValueError):
     """An error was encountered during content manifest generation."""
+
+
+class NotImplementedError(CachitoError, ValueError):
```

Comment:
I think `NotImplementedError` is a Python built-in, can you rename this to avoid confusion?

Reply:
hmm, that sounds a bit weird to me, maybe `CachitoNotImplementedError`? Or just `CachitoNotImplemented`?

Reply:
changed that to `CachitoNotImplementedError`, thanks :)